samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -0,0 +1,186 @@
+
+.. note::
+    This sample must be paired with :ref:`bluetooth_mesh_sensor_client` to show any functionality. The server provides the sensor data used by the client.
+
+Overview
+********
+
+This sample is split into the following source files:
+
+* A :file:`main.c` file to handle initialization.
+* One additional file for handling Mesh models, :file:`model_handler.c`.
+
+The following Mesh sensor types are used in this sample:
+
+* :cpp:var:`bt_mesh_sensor_present_dev_op_temp` - Published by the server according to its publishing period (see :ref:`bluetooth_mesh_sensor_server_conf_models`).
+* :cpp:var:`bt_mesh_sensor_rel_runtime_in_a_dev_op_temp_range` - Periodically requested by the client.
+* :cpp:var:`bt_mesh_sensor_presence_detected` - Published when a button is pressed on the server board.
+* :cpp:var:`bt_mesh_sensor_time_since_presence_detected` - Periodically requested by the client and published by the server according to its publishing period (see :ref:`bluetooth_mesh_sensor_server_conf_models`).
+
+Moreover, the on-chip ``TEMP_NRF5`` temperature sensor is used.
+
+Provisioning
+============
+
+The provisioning is handled by the :ref:`bt_mesh_dk_prov`.
+Use `nRF Mesh mobile app`_ for provisioning and configuring of models supported by the sample.
+
+Models
+======
+
+The following table shows the Mesh sensor server composition data for this sample:
+
+   +---------------+
+   |  Element 1    |
+   +===============+
+   | Config Server |
+   +---------------+
+   | Health Server |
+   +---------------+
+   | Sensor Server |
+   +---------------+
+
+The models are used for the following purposes:
+
+* Config Server allows configurator devices to configure the node remotely.
+* Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
+  These callbacks trigger blinking of the LEDs.
+* Mesh sensor server provides sensor data to one or more :ref:`Mesh sensor client(s) <bt_mesh_sensor_cli_readme>`.
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the ``TEMP_NRF5`` temperature sensor, and :ref:`dk_buttons_and_leds_readme` to detect button presses.
+
+Requirements
+************
+
+* One of the following development kits:
+
+  * |nRF52840DK|
+  * |nRF52DK|
+
+* Smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
+
+  * `nRF Mesh mobile app for Android`_
+  * `nRF Mesh mobile app for iOS`_
+
+* :ref:`bluetooth_mesh_sensor_client` sample application programmed on a separate device and configured according to the Mesh sensor client's :ref:`testing guide <bluetooth_mesh_sensor_client_testing>`.
+
+User interface
+**************
+
+Buttons:
+   Can be used to input the OOB authentication value during provisioning.
+   All buttons have the same functionality during this procedure.
```

Comment:
Hm, isn't Button 1 an exception here?

Reply:
All buttons have the same functionality during the provisioning procedure